<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRGBDMXSettingsEntryUi</class>
 <widget class="QWidget" name="OpenRGBDMXSettingsEntryUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>199</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>E131 settings entry</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="4">
       <widget class="QLabel" name="BrightnessLabel">
        <property name="text">
         <string>Brightness Channel:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QLineEdit" name="BrightnessEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Blue Channel:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLineEdit" name="GreenEdit"/>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="RedEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="NameLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="GreenLabel">
        <property name="text">
         <string>Green Channel:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="RedLabel">
        <property name="text">
         <string>Red Channel:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="BlueEdit"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="NameEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="KeepaliveTimeLabel">
        <property name="text">
         <string>Keepalive Time:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="KeepaliveTimeEdit"/>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="PortLabel">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="PortEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>NameEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
